### PR TITLE
Add PatrickXYS as kubeflow-testing team member

### PR DESCRIPTION
**Pull request by PatrickXYS**
Given existing Shared Test-infra has some necessary work to be done, add myself as Kanban writer to help drive development of testing work.

Here are some data points I can come up with, which should be reviewed and finished after 1.2 release:

1. Migrate from ksonnet-based Argo Workflow to Tekton
2. To migrate to Tekton, we need to install / configure Tekton on Shared Test-infra
3. Provide uniform scripts and enforce WG to use scripts, sacrifice some flexibility to obtain better availability / management.
4. Research release infra, which is CD pipeline.
5. Divide test-infra into Production and Development environment, every change need to be tested on development env first, then deploy to production. This might slow down development speed, but I think it's necessary if WG want to use shared test-infra. Otherwise, they might need to find preferred solution. 
6. Automate Webhook Management https://github.com/kubeflow/internal-acls/issues/354
7. Existing kfctl + manifests are expensive and flaky, need to be more modular and stable
8. Generating Kubeflow + Kubernetes Compatibility Matrix
9. Expose Shared Test-infra configuration
10. Testing Image CD
11. Upgrade testing image dependencies https://github.com/kubeflow/testing/issues/779
12. Two bots screaming in the same PR if user calls `/retest`

/cc @jlewi @Jeffwan @Bobgy

### CODE REVIEW

**Comment by PatrickXYS** [comment]
/cc @andreyvelich @jinchihe 

Let me know if there's more points you think we can improve

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -913,6 +913,7 @@ orgs:
             - cspavlou
             - quanjielin
             - kunmingg
+            - PatrickXYS
```
Comment (andreyvelich) [inline github-orgs/kubeflow/org.yaml:916]: Should we alphabetise it ?

**Review by andreyvelich — COMMENTED** [review]
Thanks you for the great work @PatrickXYS!
Should we also point this issue on the list: https://github.com/kubeflow/internal-acls/issues/354 ?

File: github-orgs/kubeflow/org.yaml (hunk 1 above)
Comment (PatrickXYS) [reply github-orgs/kubeflow/org.yaml:916]: Yeah let me alphabetic it

**Comment by Jeffwan** [comment]
`project-maintainers` has wide permissions on multiple repos. If you would like to do testing work, I think you may not need `project-maintainers` permission. 

If you like to manage Kanban board to track some testing infra related issues, I would suggest you to create a testing team and put `kubeflow/testing` under the management which makes more sense. Because you don't want to create tags or cut release or other repos owned by different WGs.

**Comment by PatrickXYS** [comment]
@Jeffwan Thanks, since I saw the project-maintainers description is `Team responsible for managing project kanban boards`, that's why I add myself here.

No problem, let me create `kubeflow/testing` instead

**Comment by Jeffwan** [comment]
I don't think `project-maintainers` is still the way we like to move forward anymore. Most repo already have corresponding WG to take over. They can manage kanban for themselves. There's no centralize kanban, the description means kanban for each individual projects

**Review by terrytangyuan — APPROVED** [review]
Thanks!

/lgtm

**Comment by jlewi** [comment]
> I don't think project-maintainers is still the way we like to move forward anymore.
+1 

nit: Should the group be named ".*-team".   I've been trying to think of a pattern for naming groups that are responsible for various bits (e.g. calendar, testing etc..). I think it would be good to have a common prefix; this way all such groups are grouped together in the org.yaml file as well as in GitHub completion. This follows are work group naming scheme.

Can anyone suggest a common prefix? Team seems a little bit generic.

**Comment by Jeffwan** [comment]
For calendar or testing, we can use prefix like `chore` or `misc`? Other groups will have `wg` prefix.

**Comment by Bobgy** [comment]
What do you think about?

oncall-testing
oncall-calendar

Would that sound too burdensome? But I think the idea is similar, when problems occur with these common infra, people need to contact them and there can be significant efforts implied by the name.

**Comment by terrytangyuan** [comment]
More ideas: calendar-support, test-support

**Comment by jlewi** [comment]
I like oncall-test. I think a common prefix is better then a common suffix.

File: github-orgs/kubeflow/org.yaml [hunk 2]
```diff
@@ -821,6 +821,13 @@ orgs:
             privacy: closed
             repos:
               kfctl: write
+          kubeflow-testing:
+            description: Team that will manage testing repo
+            maintainers:
+            - PatrickXYS
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml:827]: Should we add @Bobgy ? This way @Bobgy will also get notified about relevant issues and can weigh in.

Comment (PatrickXYS) [reply github-orgs/kubeflow/org.yaml]: Sounds good, added @Bobgy

**Comment by PatrickXYS** [comment]
I renamed the team name as `oncall-testing` @jlewi Can you take another look?

**Pull request merged**